tests/test_config_space_from_yaml.py

```diff
@@ -0,0 +1,362 @@
+                "type": "categorical",
+                "choices": [-1, 0, 1, 3, 5, 7, 9],
+            },
+            "classifier_dropout": {
+                "param_type": "searchable",
+                "name": "classifier_dropout",
+                "type": "discrete_uniform",
+                "low": 0.0,
+                "high": 0.5,
+                "q": 0.05,
+            },
+        }
+    )
+    return config
+
+
+@pytest.fixture()
+def lightning_classification_wrong_param_config_dict(
+    lightning_classification_config_dict: Dict[str, Any]
+) -> Dict[str, Any]:
+    config = deepcopy(lightning_classification_config_dict)
+    config["parameters"].update(
+        {"label_all_tokens": {"param_type": "constant", "name": "label_all_tokens", "value": False}}
+    )
+    return config
+
+
+@pytest.fixture()
+def base_yaml_config_file_path(
+    output_path: "TemporaryDirectory[str]", base_config_dict: Dict[str, Any]
+) -> Path:
+    output_path = Path(output_path.name).joinpath("config.yml")
+    with open(output_path, "w") as f:
+        yaml.dump(base_config_dict, f, default_flow_style=False)
+    return output_path
+
+
+@pytest.fixture()
+def flair_trainer_yaml_config_file_path(
+    output_path: "TemporaryDirectory[str]", flair_trainer_config_dict: Dict[str, Any]
+) -> Path:
+    output_path = Path(output_path.name).joinpath("flair_trainer_config.yml")
+    with open(output_path, "w") as f:
+        yaml.dump(flair_trainer_config_dict, f, default_flow_style=False)
+    return output_path
+
+
+@pytest.fixture()
+def flair_sequence_labeling_yaml_config_file_path(
+    output_path: "TemporaryDirectory[str]", flair_sequence_labeling_config_dict: Dict[str, Any]
+) -> Path:
+    output_path = Path(output_path.name).joinpath("flair_sequence_labeling_config.yml")
+    with open(output_path, "w") as f:
+        yaml.dump(flair_sequence_labeling_config_dict, f, default_flow_style=False)
+    return output_path
+
+
+@pytest.fixture()
+def lightning_classification_yaml_config_file_path(
+    output_path: "TemporaryDirectory[str]", lightning_classification_config_dict: Dict[str, Any]
+) -> Path:
+    output_path = Path(output_path.name).joinpath("lightning_classification_config.yml")
+    with open(output_path, "w") as f:
+        yaml.dump(lightning_classification_config_dict, f, default_flow_style=False)
+    return output_path
+
+
+@pytest.fixture()
+def lightning_classification_wrong_param_yaml_config_file_path(
+    output_path: "TemporaryDirectory[str]",
+    lightning_classification_wrong_param_config_dict: Dict[str, Any],
+) -> Path:
+    output_path = Path(output_path.name).joinpath(
+        "lightning_classification_no_embedding_name_config.yml"
+    )
+    with open(output_path, "w") as f:
+        yaml.dump(lightning_classification_wrong_param_config_dict, f, default_flow_style=False)
+    return output_path
+
+
+def compare_config_with_yaml(config_space: BaseConfigSpace, config: Dict[str, Any]) -> None:
```

Comment:
rename to `assert_config_to_yaml` to describe the mechanism of function in the accurate way

Reply:
done